spec/models/topics_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe 'topics' do
+
```

Comment:
Now we're validating topics, and since we're going to be using factories, it is important to make sure that the factory for topics is itself valid. So add a test something like the following:
```ruby
context 'factory' do
  subject { FactoryBot.build(:topic) }
  it { is_expected.to be_valid }
end
```